docs/docs/software-architecture.md

```diff
@@ -167,15 +166,15 @@ The user interacts with the Swapchain UI without signing-up or authenticating an
 System:   
 After the system receives the proposal submitted by the user a swap request is carried out using an HTLC. Therefore, the signatures are verified so that the private key matches the public key for the swap to take place. As soon as all required conditions are met it relays back and responds to the system to successfully transfer the funds. The involved users will receive a notification if the transaction was successful or not. In case the transaction fails, the users will receive a notification that they will be receiving the refund.
```

Comment:
we dont have the notification for refunding yet, our only notification until now is: failure and the type of error that occurs is shown.